src/Handler/StreamHandler.php

```diff
@@ -210,7 +210,7 @@ private function drain(
         Psr7\copy_to_stream(
             $source,
             $sink,
-            strlen($contentLength) > 0 ? (int) $contentLength : -1
+            (strlen($contentLength) > 0 && (int) $contentLength > 0) ? (int) $contentLength : -1
```

Comment:
Wouldn't `empty($contentLength)` do the same?

`$contentLength` comes from `getHeaderLine` which will return a string with the content length or an empty string.